content/influxdb/cloud-dedicated/admin/users/_index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Manage users
+seotitle: Manage users and permissions in InfluxDB Cloud Dedicated
+description: >
+  Manage users and access to resources in your InfluxDB Cloud Dedicated cluster.
+  Assign user groups for role-based access control and security.
+menu:
+  influxdb_cloud_dedicated:
+    parent: Administer InfluxDB Cloud
+weight: 101
+influxdb/cloud-dedicated/tags: [user groups]
+related:
+  - /influxdb/cloud-dedicated/reference/internals/security/
+  - /influxdb/cloud-dedicated/admin/tokens/ 
+---
+
+Manage users and access to resources in your {{% product-name %}} cluster.
+
+By assigning users to different groups based on the level of access they need,
+you can minimize unnecessary access and reduce the risk of inadvertent
+actions.
+User groups associate access privileges with user attributes--an important part of the
+Attribute-Based Access Control (ABAC) security model which grants access based on
```

Comment:
I'm going to copy/paste [a comment](https://docs.google.com/document/d/17m4KZct5UnLDlqqfhZ7TJel-ijEF73ikQHBtHRjkxHI/edit?disco=AAABWivDpw8) I left on [the original ABAC Phase 2 HLR](https://docs.google.com/document/d/17m4KZct5UnLDlqqfhZ7TJel-ijEF73ikQHBtHRjkxHI) for visibility:

> It's probably partially accurate to call this ABAC, since the access control is determined in part by attributes. However, since those attributes are mapped to what are effectively user roles, we may want to use a different term for the Phase 2 functionality to distinguish the identity provider attributes -> User Groups mapping from a more robust ABAC solution, especially since I'm not 100% sure we'll actually end up with ABAC by the end of this.

It might be worth clarifying that when I say "I'm not 100% sure we'll actually end up with ABAC by the end of this", I just mean that we don't currently have a design or requirements in place for ABAC. I wouldn't be surprised if we end up with RBAC + identity provider attribute mapping.

Reply:
@dsprogis Unless you know otherwise, should we remove the mention of ABAC then, given Dustin's comment?

> I wouldn't be surprised if we end up with RBAC + identity provider attribute mapping.

Reply:
While Dustin may be correct, the product direction has been ABAC so let's stick with that for now.
If we end up settling with RBAC++ then we can articulate that later.